src/current/_includes/releases/v24.1/v24.1.2.md

```diff
@@ -0,0 +1,71 @@
+- Fixed a bug where a change to a [user-defined type](https://www.cockroachlabs.com/docs/v24.1/create-type) could cause queries against tables using that type to fail with an error message like: `"histogram.go:694: span must be fully contained in the bucket"`. The change to the user-defined type could come directly from an [`ALTER TYPE`](https://www.cockroachlabs.com/docs/v24.1/alter-type) statement, or indirectly from an [`ALTER DATABASE ... ADD REGION`](https://www.cockroachlabs.com/docs/v24.1/alter-database#add-region) or [`DROP REGION`](https://www.cockroachlabs.com/docs/v24.1/alter-database#drop-region) statement (which implicitly changes the `crdb_internal_region` type). This bug had existed since UDTs were introduced in v20.2. [#124810][#124810]
+- Fixed a bug where [telemetry logs](https://www.cockroachlabs.com/docs/v24.1/logging#telemetry) had the same [statement fingerprint](https://www.cockroachlabs.com/docs/v24.1/ui-statements-page#sql-statement-fingerprints) ID for different SQL statements. [#125000][#125000]
+- Fixed an issue where [adding a column](https://www.cockroachlabs.com/docs/v24.1/alter-table#add-column) with a default value of an empty [array](https://www.cockroachlabs.com/docs/v24.1/array) would not succeed. [#125325][#125325]
+- [`ALTER TABLE ... ADD CONSTRAINT UNIQUE`](https://www.cockroachlabs.com/docs/v24.1/alter-table) will now fail with a well-formed error message and code `42601` if a statement tries to add a [unique constraint](https://www.cockroachlabs.com/docs/v24.1/unique) on an expression. [#125417][#125417]
+- Fixed a bug where the [`public` schema](https://www.cockroachlabs.com/docs/v24.1/schema-design-overview#schemas) would be created with the wrong owner. Previously the [`admin` role](https://www.cockroachlabs.com/docs/v24.1/security-reference/authorization#roles) would own the `public` schema. Now, the database owner is also the owner of the `public` schema. The ownership can be altered after the schema is created. [#125533][#125533]
+- Fixed a bug in v24.1, v23.2, and v23.1 where using the `changefeed.aggregator.flush_jitter` [cluster setting](https://www.cockroachlabs.com/docs/v24.1/cluster-settings#setting-changefeed-aggregator-flush-jitter) with the [`min_checkpoint_frequency`](https://www.cockroachlabs.com/docs/v24.1/create-changefeed#min-checkpoint-frequency) option set to `0` could cause panics. [#125459][#125459]
+- Resolved a concerning log message that said `"expiration of liveness record ... is not greater than expiration of the previous lease ... after liveness heartbeat"`. This message is no longer possible. [#125449][#125449]
```

Comment:
```suggestion
- The log message `"expiration of liveness record ... is not greater than expiration of the previous lease ... after liveness heartbeat"` is no longer generated. [#125449][#125449]
```

Reply:
yes that language seems much less concern-inducing :-)